### PR TITLE
Update launch to match the repo

### PR DESCRIPTION
**Pull request by RaananW**
This PR updates launch.json to match the current tools we are using.

WebGL2 and WebGPU tests have moved to playwright. Both all and single test are supported. Note that I am still stronglyr ecommend to use the playwright vscode extension instead of running tests like that, as it is simpler to evaluate, easier to use, and faster.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15957/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15957/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15957/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15957/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15957/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15957/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15957/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15957/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15957/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15957/merge/testResults/webgpuplaywright/index.html

File: .vscode/launch.json [hunk 1]
```diff
@@ -284,56 +284,28 @@
...
-            Launch the playground to be use to test changes in dev packages
-            */
-            "type": "msedge",
-            "request": "launch",
-            "name": "Run and Watch Dev host - SSL (ES6-Dev)",
-            "url": "https://localhost:1338",
-            "outFiles": ["!**/node_modules/**"],
-            "preLaunchTask": "Run and Watch Dev host (Dev)"
-        },
+        // {
```
Comment (sebavan) [inline .vscode/launch.json:287]: should we not just remove them ?

**Review by sebavan — APPROVED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (RaananW) [reply .vscode/launch.json:287]: Of course! Actually meant to do it here. I'll take care of that later

**Pull request merged**